api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -65,7 +65,7 @@ public void getCohort_shouldOnlyGetNonVoidedCohortsByName() throws Exception {
 		// now do the actual test: getCohort by name and expect a non voided cohort
 		Cohort exampleCohort = service.getCohort("Example Cohort");
 		assertNotNull(exampleCohort);
-		assertEquals(2, exampleCohort.size());
+		assertEquals(1, exampleCohort.size());
```

Comment:
No, the test data has only one non-voided cohort. Not sure why it was 2 earlier:

```
  <cohort cohort_id="1" name="Example Cohort" description="First cohort that is now voided" creator="1" date_created="2005-01-01 00:00:00.0" voided="true" voided_by="1" date_voided="2005-01-01 00:00:00.0" void_reason="This is voided" uuid="h9a9m0i6-15e6-467c-9d4b-mbi7teu9lf0f"/>
  <cohort cohort_id="2" name="Example Cohort" description="Second cohort that has not been voided" creator="1" date_created="2005-01-01 00:00:00.0" voided="false"/>
```